Dockerfile

```diff
@@ -1,37 +1,12 @@
-####################################################################################################
-# Build binaries
-####################################################################################################
-FROM golang:1.16.6 as build
-
-WORKDIR /go/src/github.com/argoproj-labs/argo-cloudops
-
-COPY go.mod ./
-COPY go.sum ./
-
-RUN go mod download
-
-COPY internal ./internal
-COPY service ./service
-COPY cli ./cli
-COPY Makefile ./
-
-COPY .git ./.git
-
-RUN make
-
-####################################################################################################
-# Run image
-####################################################################################################
 FROM debian:latest
 
+ARG BINARY
```

Comment:
This enables the `Dockerfile` to not be specific to quickstart (as the local file location is passed in). If/when we produce "official" images, this can be revisited.